docs/tutorial/index.rst

```diff
@@ -0,0 +1,62 @@
+Javascript Sandbox Tutorial
+===========================
+
+This is a javascript sandbox tutorial for writing Vumi Go Application.
+
+What is sandbox?
+----------------
+
+A sandbox is a type of software testing environment that enables the isolated execution of software or programs for independent evaluation, monitoring or testing. Basically sandboxing is to test a software in a controlled environment before putting it to actual use.
```

Comment:
This is a slight misunderstanding of the meaning of "sandbox" in this context (although it might be true in other contexts). Here the sandbox is also an isolated execution environment, but its used in production, not testing, and its role is to provide access to carefully selected external resources and capabilities (e.g. logging, web requests, a key-value store, sending messages).